template.feature

```diff
@@ -121,31 +123,75 @@ I should see the image <context> "<value>" is enabled|disabled
 	eg: I should see the image with id "profile-image" is enabled
 
 #Menu navigation:
-# Check for elements
-# open menu
-# close menu
-# click on menu option
-# Checkbox/Radio button
-# Enabled
-# Disabled
+ #This is a complex object built from the atomic steps. 
+
+Checkbox/Radio button:
+Visibility
+I see the "<element_name>" <type> <context> "<value>" is <condition>
+Examples:
+	| type         | condition    | context    |
+	| checkbox     | enabled      | in section |
+	| radio button | disabled     | with id    |
+	|              | selected     |            | 
+	|              | not selected |            |
+
+Selection	
+I <selection> the "<element_name>" <type> <context> "<value>"
+Examples:
+	| selection | type        | context    | 
+	| select    | checkbox    | in section |
+	| unselect  | radiobutton | with id    |
 # select
 # unselect
-# Slider
-# current value
-# increase
-# decrease
-# Dialog boxes
-# dialog box shown
-# dialog box hidden
-# Lists
-# Should see "x" in "y" row "z" column
-# number of elements
-# Page scroll
-# Scroll up
-# Scroll down
-# Elements (eg: alert icon)
-# Check if elements exist
-# check if elements don’t exist
+Slider:
+current value
+I see the "<slider_name>" slider <context> "<value>" has value "<value>"
+
+increase/decrease
+I <option> the "<slider_name>" slider <context> "<value>"'s value to "<value>"
+
+Examples:
+	| context    | option   |
+	| in section | increase |
+	| with id    | decrease |
+	eg. I see the "rating" slider with id "rating-slider" has value "4"
+	eg. I increase the "rating" slider with id "rating-slider"'s value to "10"
```

Comment:
yeah I agree. the sentence formation seems better the way you've mentioned it. 

Increase decrease would map to the values right? You slide the slider to the right or left, to increase or decrease. But that wouldn't cover vertical sliders.